src/documentation/reference/ReferenceNode.tsx

```diff
@@ -77,9 +77,10 @@ const ReferenceNode = ({ anchor, docs, ...props }: ApiDocEntryNodeProps) => {
         fontSize="sm"
         spacing={3}
         p={5}
+        pb={3}
```

Comment:
The reason the reference entries look off is because they start with a code section with no spacing above it, and end with text, or a 'show more' button that both have some inherent spacing below them (the button more than plain text). Reducing the padding here helps counter that, but a bit part of me doesn't like uneven padding.